robottelo/config/validators.py

```diff
@@ -6,7 +6,15 @@
 
 validators = dict(
     server=[
-        Validator("server.hostname", must_exist=True),
+        Validator("server.hostname", must_exist=False),
+        Validator("server.hostnames", must_exist=True, is_type_of=list),
+        Validator("server.version.release", must_exist=True),
```

Comment:
I question this being must_exist=True.

If you have a host, it should just get pulled from that in order to deploy more sat/cap instances. 

It's only required if you're starting with no hosts and using on-demand, and even then could be considered optional, as you can call the workflows without providing any release version or snap.

Reply:
This is more so required for bugzilla checks than satellite checkouts.